Automattic/simplenote-macos
# Debt: Initial Setup Flow

### Fix
In this PR we're porting over to Swift the Welcome Note flow, and wiring Options usage (NSUserDefaults wrapper).

cc @eshurakov Slowly swifting the AppDelegate!

Thank you sir!!

### Test: First Install
1. Delete the macOS Welcome Note from your Simplenote account
2. Make sure you empty the Trash bin
3. In ZSH run `defaults delete com.automattic.SimplenoteMac.beta SPFirstLaunch`
4. Relaunch the app

- [x] Verify the Welcome Note shows up and it's properly localized in your local machine's language

### Test: Deletion
1. Launch the app
2. Delete the macOS Welcome Note from your Simplenote account
3. Make sure you empty the Trash bin
4. **Relaunch!**

- [x] Make sure the **Welcome Note** does not show up again!

### Release
These changes do not require release notes.
